### ISSUE TITLE
Error (Kiwi Browser)

### ISSUE THREAD

**Issue by Saicobo** [Tobi823/ffupdater#252]
Stacktrace:
```
de.marmaro.krt.ffupdater.background.UnrecoverableBackgroundException: The background job fails due to an unrecoverable exception.
	at de.marmaro.krt.ffupdater.BackgroundJob$doWork$2.invokeSuspend(Unknown Source:53)
	at w3.a.resumeWith(Unknown Source:8)
	at j4.i0.run(Unknown Source:100)
	at n4.a$a.run(SourceFile:82)
Caused by: de.marmaro.krt.ffupdater.network.exceptions.InvalidApiResponseException: can't find release after all tries - abort
	at de.marmaro.krt.ffupdater.network.github.GithubConsumer.updateCheck(SourceFile:2)
	at de.marmaro.krt.ffupdater.network.github.GithubConsumer$updateCheck$1.invokeSuspend(Unknown Source:19)
	at w3.a.resumeWith(Unknown Source:8)
	at j4.i0.run(Unknown Source:100)
	at m4.d.run(Unknown Source:11)
	at n4.j.run(Unknown Source:2)
	... 1 more
```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 77.3.0 (114) release |
| Device | SM-A307G (a30sub, a30s, exynos7904) |
| Manufacturer | samsung (samsung) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 11 (SDK: 30) |
| OS | 21DJ6616, dpi, release-keys, 1654025639000 |

**Comment by Tobi823** [comment]
Fixed version (77.3.1) is now released

**Comment by Theluga** [comment]
It's not working. 
Ffupdater only shows the numerical build of post. And does show the latest build. 
Or You download directly from github or update through play store.
I thought it was cache. When I cleared storaged. It tried to install an older version and failed ofc.

**Comment by Tobi823** [comment]
There was a new bug which is fixed in 77.3.3
